### PR TITLE
add Cubyz🗒️Voxel sandbox game

### PR DESCRIPTION
**Pull request by Zig-Rust-Odin**

### CODE REVIEW

File: README.md
Comment (C-BJ) [inline README.md:1]: Please arrange the projects in alphabetical order, `C` should come after the number `4`.

**Review by C-BJ — CHANGES_REQUESTED** [review]

File: README.md
Comment (Zig-Rust-Odin) [reply README.md:1]: You just edit and merge.

Comment (C-BJ) [reply README.md:1]: I seem unable to modify your fork.

Comment (C-BJ) [reply README.md:1]: Sorry, my bad.

**Comment by C-BJ** [comment]
Thanks.

**Pull request merged**